src/System.Text.Json/experiments/Json.Net.Test/Json.Net.Tests/JsonParser.cs

```diff
@@ -0,0 +1,565 @@
+﻿using System;
+
+namespace Json.Net.Tests
+{
+    public struct JsonParser : IDisposable
```

Comment:
It was there to help with performance analysis when running the profiler so you get more details. It is essentially a copy of what is found in src.